plugin.dbmc/addon.xml

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.dbmc" name="Dbmc (Dropbox add-on)" version="0.9.3"
+<addon id="plugin.dbmc" name="Dbmc (Dropbox add-on)" version="0.9.5"
 	provider-name="Joost Kop">
 	<requires>
 		<import addon="xbmc.python" version="2.1.0" />
```

Comment:
Please update to teh correct xbmc.python version for Helix - 2.19.0. Please see http://kodi.wiki/view/Addon.xml#Dependency_versions